semeio/workflows/correlated_observations_scaling/cos.py

```diff
@@ -12,6 +12,117 @@
 from semeio.workflows.correlated_observations_scaling.obs_utils import keys_with_data
 
 
+_DESCRIPTION = """
+    Correlated_observation_scaling is an ERT workflow job that does Principal
+    Component Analysis (PCA) scaling of observations in ERT. The job accepts a
+    configuration as the only argument.
+
+    A valid job configuration is a YAML file in one of three formats:
+    a CALCULATE_KEYS and UPDATE_KEYS pair; a list of such pairs; or a singular
+    CALCULATE_KEYS directive. Each directive accepts a list of keys that
+    each are a key and index hash. Keys can contain wildcards, and indices
+    can represent ranges like "1-10,11,12". No index value implies all
+    values. CALCULATE_KEYS can be configured using threshold which sets the
+    threshold for PCA scaling [default: 0.95]; std_cutoff, the cutoff for
+    standard deviation filtering [defaults to ``ert`` values]; and alpha for filtering
+    between ensemble mean and observations [defaults to ``ert`` values].
+
+Correlated Observations Scaling Configuration
+=============================================
+{}
+""".format(
+    configsuite.docs.generate(job_config.build_schema())
+)
+
+
+_EXAMPLES = """
```

Comment:
I mean.. Not sure if it is an overkill.. But if we split the examples out into individual strings, could we then create tests in order to ensure that the examples are in fact valid?

Reply:
I should be able to check that they are valid configs, but the context validation will be missing. Might be overkill, but you never know, good to have forced up to date docs.